### DISCUSSION TITLE
Pico2/v6 timeline

### DISCUSSION THREAD

**Discussion by eryjus** [gusmanb/logicanalyzer#161] [Q&A]
I am quite interested in getting up and running with this project as it is far more mature than my own similar effort.  

That said, there is some hardware I need to purchase no matter what version I run.  Which now leads to the question on whether v6 is close or if I should stick with v5?

I run everything on Linux so I would need to work out the build steps. I realize I would be blazing a trail.  But I am willing to take on the additional work.  

I'm just looking for some guidance/recommendations given today's status.  

Thanks in advance.

**Comment by gusmanb** [comment]
Target is to release the V6.0 this month.
